### PR TITLE
fixes #64: Increase MAX_CONTENT_LENGTH

### LINKED ISSUE #64: Increase content length for uploads
@amirardalan9473 flagged that he cannot upload a file with 43mb size. We initially thought it was related to the ingress controller that hosts the backend but looking at the logs from the backend we see that its the web server that is giving the issue. 

Screenshot from nachet-backend logs (thank you argoCD) : 
image
Looking at the code, I see we are using Quart as a web framework. Parsing default values i see that the limit for content is set as 16 mb : 
image
reference :
https://github.com/pallets/quart/blob/8a0cdd5a650c29d036dfec3000818b0e06ce3e0c/src/quart/config.py#L19

We would need to increase this value to at least 200mb by doing something like : 
```
from flask import Flask, request

app = Flask(__name__)

app.config['MAX_CONTENT_LENGTH'] = 200 * 1024 * 1024
```

### PR DESCRIPTION
**Pull request by MaxenceGui**
- [x] Add NACHET_MAX_CONTENT_LENGTH environment variable
- [x] Config the MAX_CONTENT_LENGTH value from the app.
- [x] Update read me to include and explain the new environment variable

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -6,3 +6,4 @@ NACHET_SUBSCRIPTION_ID=
 NACHET_RESOURCE_GROUP=
 NACHET_WORKSPACE=
 NACHET_MODEL=
+NACHET_MAX_CONTENT_LENGHT=
```
Comment (SonOfLope) [inline .env.template]:
```suggestion
NACHET_MAX_CONTENT_LENGTH=
```

File: README.md [hunk 2]
```diff
@@ -37,11 +37,12 @@ python -m unittest discover -s tests
 Start by making a copy of `.env.template` and renaming it `.env`. For the backend to function, you will need to add the missing values:
 
 * **NACHET_AZURE_STORAGE_CONNECTION_STRING**: Connection string to access external storage (Azure Blob Storage).
-* **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed model for inferencing. 
+* **NACHET_MODEL_ENDPOINT_REST_URL**: Endpoint to communicate with deployed model for inferencing.
 * **NACHET_MODEL_ENDPOINT_ACCESS_KEY**: Key used when consuming online endpoint.
 * **NACHET_DATA**: Url to access nachet-data repository
 * **NACHET_HEALTH_MESSAGE**: Health check message for the server.
+* **NACHET_MAX_CONTENT_LENGHT**: Set the maximum size of the file that can be uploaded to the backend. Need to be the same size as the `proxy body size` variable from DevSecOps.
```
Comment (SonOfLope) [inline README.md]:
```suggestion
* **NACHET_MAX_CONTENT_LENGTH**: Set the maximum size of the file that can be uploaded to the backend. Needs to be the same size as the `client_max_body_size` [value](https://github.com/ai-cfia/howard/blob/dedee069f051ba743122084fcb5d5c97c2499359/kubernetes/aks/apps/nachet/base/nachet-ingress.yaml#L13) set from the deployment in Howard.
```

Comment (SonOfLope) [reply README.md]: The hyperlink doesn't seem to work in suggested changes but I updated the variable since it wasn't the right one.

**Review by SonOfLope — CHANGES_REQUESTED** [review]

**Review by SonOfLope — APPROVED** [review]

**Review by k-allagbe — APPROVED** [review]

**Review by RussellJimmies — APPROVED** [review]

File: app.py [hunk 3]
```diff
@@ -52,6 +52,8 @@
 
 app = Quart(__name__)
 app = cors(app, allow_origin="*", allow_methods=["GET", "POST", "OPTIONS"])
+# Using eval as the env variable will likely be 200 * 1024 * 1024
+app.config["MAX_CONTENT_LENGTH"] = eval(os.getenv("NACHET_MAX_CONTENT_LENGTH"))
```
Comment (rngadam) [inline app.py]: don't use eval:

https://www.codiga.io/blog/python-eval/

also here a simple int() to convert from string to int would do?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: app.py (hunk 3 above)
Comment (MaxenceGui) [reply app.py]: Done

**Review by rngadam — APPROVED** [review]

**Comment by rngadam** [comment]
don't forget to change your description to ```fixes #64```

**Pull request merged**